<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaskExtractClass</class>
 <widget class="QDialog" name="MaskExtractClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaskExtract</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>396</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>输入栅格：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="inputCombo">
        <property name="minimumSize">
         <size>
          <width>271</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>271</width>
          <height>21</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="inputFileBtn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="TigerGIS.qrc">
          <normaloff>:/TigerGIS/res/folderOpen.png</normaloff>:/TigerGIS/res/folderOpen.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>输入掩膜：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="maskCombo">
        <property name="minimumSize">
         <size>
          <width>271</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>271</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="maskFileBtn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="TigerGIS.qrc">
          <normaloff>:/TigerGIS/res/folderOpen.png</normaloff>:/TigerGIS/res/folderOpen.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>输出栅格：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outputEdit">
        <property name="minimumSize">
         <size>
          <width>271</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>271</width>
          <height>21</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="outputFileBtn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="TigerGIS.qrc">
          <normaloff>:/TigerGIS/res/folderOpen.png</normaloff>:/TigerGIS/res/folderOpen.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>320</y>
     <width>301</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="cancelBtn">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="conductBtn">
      <property name="text">
       <string>运行</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TigerGIS.qrc"/>
 </resources>
 <connections/>
</ui>
